pylint/checkers/variables.py

```diff
@@ -73,7 +73,11 @@
 )
 
 
-class VariableVisitConsumerAction(Enum):
+def register(linter: PyLinter) -> None:
```

Comment:
The other checkers seem to all place `register()` at the end of the file, so I suggest keeping that pattern here.